packages/snjs/lib/services/api/http_service.ts

```diff
@@ -98,6 +111,19 @@ export class SNHttpService extends PureService {
     return request;
   }
 
+  private versionHeaderName(): string {
+    switch (this.environment) {
+      case Environment.Web:
+        return 'X-Web-Version';
+      case Environment.Mobile:
+        return 'X-Mobile-Version';
+      case Environment.Desktop:
+        return 'X-Desktop-Version';
+      default:
+        throw Error(`'${this.environment}' is not correct environment value`);
+    }
+  }
+
```

Comment:
I'm not sure on the backend requirements, @karolsojko correct me if I'm wrong, but wouldn't it be better to have a single header name, and the header value would contain something like `Web: x.y.z`?

Searching/reporting from the logs for stuff wouldn't require us to have a complex query string like `"X-Web" OR "X-Mobile"...

Reply:
it's common for a lot of servers to ditch custom headers that are not `X-...` prefixed. Event though there's an RFC of deprecating the `X-...` naming convention - if we want to add custom defined headers we have to keep them in the `X-...` to be compatible with what's out there I think

Reply:
I think I didn't explain myself. I was proposing of having a single header name instead of three different header names.

Reply:
hmm let's say we have web at version 1.2.3 and mobile at version 1.2.3 how would we know if the request is coming from a mobile client or web client?

Reply:
`X-Version: Web-1.2.3` vs `X-Version: Android-1.2.3`?

Reply:
Good idea. @vardan-arm so let's have `X-SNJS-Version` and `X-Application-Version`

Reply:
Good point @gorjan5sk , updated.